back/app/controllers/web_api/v1/action_descriptors_controller.rb

```diff
@@ -5,7 +5,7 @@ class WebApi::V1::ActionDescriptorsController < ApplicationController
   skip_after_action :verify_authorized, only: [:initiatives]
 
   def initiatives
-    descriptors = Permissions::InitiativePermissionsService.new.action_descriptors(current_user)
+    descriptors = Permissions::InitiativePermissionsService.new(current_user).action_descriptors
```

Comment:
Moving user to the instance level is definitely a good thing. I nearly did this change myself in the last tandem - but ran out of time!